src/main/java/reposense/parser/CsvParser.java

```diff
@@ -171,9 +176,9 @@ protected String getOrDefault(final CSVRecord record, String header, String defa
     }
 
     /**
-     * Returns the value of {@code record} at the column with the header {@code header} as a {@code List},
+     * Returns the value of {@code record} at the column with the header {@code header} as a {@link List},
```

Comment:
I think this line can end with a full stop. The delimiter and empty list behaviour can be in follow-up sentences.